torchrl/objectives/td3.py

```diff
@@ -331,66 +330,484 @@ def in_keys(self, values):
     def _cached_detach_qvalue_network_params(self):
         return self.qvalue_network_params.detach()
 
-    @property
-    @_cache_values
-    def _cached_stack_actor_params(self):
-        return torch.stack(
-            [self.actor_network_params, self.target_actor_network_params], 0
+    def actor_loss(self, tensordict):
```

Comment:
Should we @dispatch?

Reply:
mmh it would mean defining in and out keys for each function right.

It might make sense so that it allow understanding what keys each function should access, but it might clutter the code and if the forward is already dispatched it is only useful when mixing losses